lib/siwapp_web/live/customer_live/edit.html.heex

```diff
@@ -1,10 +1,33 @@
 <h1> <%= @page_title %> </h1>
-
-<.form let={f} for={@customer} phx-submit="save">
+<.form let={f} for={@changeset} phx-submit="save">
```

Comment:
Do not change this file at all. This is not your business in this PR

Reply:
Again, won't happen then